components/Footer.tsx

```diff
@@ -0,0 +1,159 @@
+      href: "/documentation",
+      items: [
+        { name: "Oscar", href: "https://docs.ccv.brown.edu" },
+        {
+          name: "Stronghold",
+          href: "/services/computing#data-risk-level-3-computing-(stronghold)",
+        },
+        { name: "Globus", href: "/services/file-storage-and-transfer#globus" },
+        { name: "Jupyter Hub", href: "/services/classroom#jupyterhub" },
+      ],
+    },
+    ourWork: {
+      name: "Our Work",
+      href: "/our-work",
+      items: [
+        { name: "Collaborations", href: "#" },
+        {
+          name: "Software",
+          href: "/our-work/software",
+        },
+        {
+          name: "Workshops and Talks",
+          href: "/our-work/workshops-and-talks",
+        },
+        {
+          name: "Publications",
+          href: "https://publications.ccv.brown.edu",
+        },
+      ],
+    },
+    help: {
+      name: "Help",
+      href: "/help",
+      items: [
+        { name: "Submit a Ticket", href: "mailto:support@ccv.brown.edu" },
+        {
+          name: "Discourse",
+          href: "https://ask.cyberinfrastructure.org/c/brown-research-computing/37",
+        },
+        {
+          name: "Slack",
+          href: "https://ccv-share.slack.com/signup#/",
+        },
+        {
+          name: "Office Hours",
+          href: "https://events.brown.edu/ccv/month",
+        },
+      ],
+    },
+    about: {
+      name: "About",
+      href: "/about",
+      items: [
+        { name: "Mission", href: "/about#mission" },
+        {
+          name: "Office of Information Technology",
+          href: "/about#office-of-information-technology",
+        },
+        {
+          name: "Our Teams",
+          href: "/about#our-teams",
+        },
+        {
+          name: "People",
+          href: "/about#people",
+        },
+        {
+          name: "Opportunities",
+          href: "/about#opportunities",
+        },
+        {
+          name: "Events",
+          href: "https://events.brown.edu/ccv/month",
+        },
+        {
+          name: "Facilities Statement",
+          href: "/about#facilities-statement",
+        },
+        {
+          name: "Diversity Statement",
+          href: "/about#diversity-statement",
+        },
+      ],
+    },
+  }
+  return (
+    <footer aria-labelledby="footer-heading" className="w-full bg-neutral-50">
+      <h2 id="footer-heading" className="sr-only">
+        Footer
+      </h2>
+      <div className="mx-auto max-w-7xl px-6 pb-8 pt-8 sm:pt-24 lg:px-8 lg:pt-16">
+        <div className="xl:grid xl:grid-cols-2 xl:gap-8">
+          <div className="space-y-8">
+            <div className="flex w-2/3 gap-3.5">
+              <LogoBrown />
+              <LogoCcv />
+            </div>
+          </div>
+
+          <div className="mt-8 grid grid-cols-1 lg:grid-cols-5 gap-8 xl:col-span-2">
+            {Object.entries(navigation).map(([_, contents]) => {
+              return (
+                <div key={_}>
```

Comment:
I personally don't love using the `_` here, it's a little nitpicky, but if you're going to use the value I think you should either change the `_` to `key` or something or use the `href` for the key value. So it could look like either of these:
 - `<div key={key> ...`
 - `<div key={href}> ...`